### PR TITLE
Dat 4080

### PR DESCRIPTION
**Pull request by nikodamn**
Invoke default navigation bar view for currently running navigation experiment on Mercury explicitly by URL.

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/elements/common/Navigate.java [hunk 1]
```diff
@@ -9,6 +9,12 @@
 
   private WebDriver driver;
 
+  /**
+   * Adding default testing group is invoked explicitly in URL is temporary change until
+   * the end of experiment (19.04.2016). Experiment is described in DAT-4052.
+   */
+  private String defaultTestingGroup = "AbTest.FAN_KNOWLEDGE_MERCURY_GLOBAL_NAV=DEFAULT";
```
Comment (drets) [inline src/test/java/com/wikia/webdriver/elements/common/Navigate.java]: `private static final` ?

File: src/test/java/com/wikia/webdriver/elements/common/Navigate.java [hunk 2]
```diff
@@ -17,7 +23,7 @@ public Navigate toPage(String pageName) {
     String host = UrlBuilder.getHostForWiki();
     String cacheBuster = pageName.equals("") || pageName.equals("/") ? "" : "?cb=" + DateTime.now().getMillis();
 
-    driver.get("http://" + host + pageName + cacheBuster);
+    driver.get("http://" + host + pageName + cacheBuster + "&" + defaultTestingGroup);
```
Comment (drets) [inline src/test/java/com/wikia/webdriver/elements/common/Navigate.java]: Did you think about using [appendQueryStringToURL](tests/blob/master/src/test/java/com/wikia/webdriver/common/core/url/UrlBuilder.java#L71) method for appending URL params to URL?

File: src/test/java/com/wikia/webdriver/elements/common/Navigate.java [hunk 3]
```diff
@@ -9,6 +9,12 @@
 
   private WebDriver driver;
 
+  /**
+   * Adding default testing group is invoked explicitly in URL is temporary change until
```
Comment (drets) [inline src/test/java/com/wikia/webdriver/elements/common/Navigate.java]: 👍

File: src/test/java/com/wikia/webdriver/elements/common/Navigate.java [hunk 4]
```diff
@@ -9,6 +9,12 @@
...
+   * the end of experiment (19.04.2016). Experiment is described in DAT-4052.
+   */
+  private static final String defaultTestingGroup = "AbTest.FAN_KNOWLEDGE_MERCURY_GLOBAL_NAV=DEFAULT";
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/elements/common/Navigate.java]: MAJOR Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AS00115)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue:
* MAJOR 1 major

Watch the comments in this conversation to review them.

File: src/test/java/com/wikia/webdriver/elements/common/Navigate.java (hunk 1 above)
Comment (nikodamn) [inline src/test/java/com/wikia/webdriver/elements/common/Navigate.java]: Good catch, thanks!

File: src/test/java/com/wikia/webdriver/elements/common/Navigate.java (hunk 2 above)
Comment (nikodamn) [inline src/test/java/com/wikia/webdriver/elements/common/Navigate.java]: As it's only temporary change I used the quicker way to do it. In general that might be a good idea, but I'd like to investigate it when we will turn off experiment and delete that parameter from URL.

Comment (drets) [inline src/test/java/com/wikia/webdriver/elements/common/Navigate.java]: Fair enough!

**Comment by drets** [comment]
LGTM :+1:

**Pull request merged**